src/RNCamera.js

```diff
@@ -53,7 +55,7 @@ type EventCallbackArgumentsType = {
   nativeEvent: Object,
 };
 
-type PropsType = typeof View.props & {
+type PropsType = ViewStyleProp & {
```

Comment:
not the correct Flow type. This is for a style prop. This needs to be the Flow type for all View Props.

Reply:
this one https://github.com/facebook/react-native/blob/master/Libraries/Components/View/ViewPropTypes.js

Reply:
`ViewProps `